### PR TITLE
advent-2015: added GopherCon 2016 announcement

### PR DESCRIPTION
**Pull request by davecheney**
screenshot from 2015-12-09 16 02 58

### CODE REVIEW

**Comment by davecheney** [comment]
@bketelsen @erikstmartin i added my name to the announcement. If you want to remove it and make it your announcement that is A-OK with me (and probably easier all things considered)

File: content/advent-2015/announcing-gophercon-2016.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+author = ["Brian Ketelsen", "Erik St Martin", "David Cheney"]
+date = "2015-12-11T08:00:00+00:00"
+title = "Announcing GopherCon 2016"
+series = ["Advent 2015", "GopherCon 2016"]
++++
+<div style="text-align:center;">
+<img src="/postimages/advent-2015/gophercon2016.svg" alt="GopherCon 2015" style="width: 300px;margin: 0 auto;"/>
+</div>
+
+GopherCon 2016 will take place at the Denver Convention Center in Denver, Colorado on July 11th through July 13th, 2016.
```
Comment (nathany) [inline content/advent-2015/announcing-gophercon-2016.md]: do you mean "Colorado Convention Center"?

File: content/advent-2015/announcing-gophercon-2016.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+Questions at tech conferences have a poor reputation.
+There’s the pressure of coming up with a coherent question while you’re racing to be the first in line for the mic.
+There’s the danger of that “well, actually...” moment.
+And it’s hard to fight the feeling that question time is just a filler while the next presenter is getting their laptop to work with the projector.
+In fact, many speakers have told us that the most stressful part of giving a presentation at a conference _is_ question time.
+
+We’re going to try something different for GopherCon 2016.
+Instead of a few minutes of question time at the end of each plenary session, we will be asking everyone to hold questions until after lunch, when we will invite speakers from the day to join an extended panel discussion.
+
+For the afternoon tutorial sessions, we expect the smaller rooms size to be more conducive to questions from the floor and encourage speakers plan their talks accordingly.
```
Comment (nathany) [inline content/advent-2015/announcing-gophercon-2016.md]: speakers <ins>to</ins> plan

File: content/advent-2015/announcing-gophercon-2016.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+Everyone is welcome at GopherCon.
+Whether you’re speaking on stage or listening from the audience, whether it's your first Go conference or your tenth, you are welcome.
+
+We strongly encourage diversity at our events and actively solicit proposals from a variety of potential speakers.
+The conference reserves the right to invite speakers who have not submitted proposals in order to create a conference that is more enjoyable, more informative, and more inclusive.
+
+# Code of Conduct
+GopherCon is proud to announce we have adopted the [Go Community Code of Conduct](https://golang.org/conduct).
+
+GopherCon, and its organising body, Gopher Academy, is dedicated to providing a harassment-free conference for everyone, regardless of age, disability, gender, nationality, race, religion, sexuality, or similar personal characteristic.
```
Comment (nathany) [inline content/advent-2015/announcing-gophercon-2016.md]: "organizing" is the more common spelling in the U.S.

**Comment by nathany** [comment]
The changes to the conference sound great. Super excited.

Thanks for preparing the announcement @davecheney.

File: content/advent-2015/announcing-gophercon-2016.md (hunk 1 above)
Comment (davecheney) [inline content/advent-2015/announcing-gophercon-2016.md]: Yup, I guessed and guessed wrong. Fixed.

File: content/advent-2015/announcing-gophercon-2016.md (hunk 2 above)
Comment (davecheney) [inline content/advent-2015/announcing-gophercon-2016.md]: fixed

File: content/advent-2015/announcing-gophercon-2016.md (hunk 3 above)
Comment (davecheney) [inline content/advent-2015/announcing-gophercon-2016.md]: I'll change them both to eu-UK

**Pull request merged**